### ISSUE TITLE
If you label some issues, label them all!

### ISSUE THREAD

**Issue by Happypig375** [jiggzson/nerdamer#183]
Assuming 'bug' is incorrect outcome, 'enhancement' is extending/adding features, and 'question' for not-bug-nor-enhancements.

A quick analysis on all open issues:

Issue|Label|Labelled?|Additional actions (if different from solving the issue)
-|-|-|-
#84|bug|No|Close and move to #181 after work has started there
#117|bug|No|Close as no response
#119|enhancement|Yes|None
#121|enhancement|No|None
#122|enhancement|No|None
#123|bug|Yes|None
#124|bug|Yes|None
#126|bug|Yes|None
#127|bug|No|None
#128|bug|No|None
#129|enhancement|No|None
#130|bug|No|None
#131|enhancement|No|None
#133|bug|No|None
#134|enhancement|No|None
#135|enhancement|No|Close as added after adding tests
#136|bug|No|Close as invalid (also label as invalid)
#137|bug|No|None
#138|question|No|Close as answered
#139|bug|No|None
#140|enhancement|No|None
#141|enhancement|No|None
#143|enhancement|Yes|None
#144|enhancement|No|None
#146|bug|No|None
#147|enhancement|No|None
#148|enhancement|No|None
#149|enhancement|No|Close as added after adding tests
#150|enhancement|Yes|Close as added after adding tests
#151|bug|No|None
#153|enhancement|No|None
#154|enhancement|No|None
#155|enhancement|Yes|None
#156|question|No|None
#158|bug|Yes|Relabel from question to bug because it is a bug
#160|enhancement|No|Close as added after adding tests
#161|bug|Yes|None
#163|enhancement|No|None
#164|bug|Yes|None
#165|bug|No|None
#166|enhancement|No|None
#167|enhancement|No|None
#170|question|No|None
#181|enhancement|Yes|None
#182|enhancement|No|None

Please change all the above Noes to Yeses.

Aside from using existing labels, maybe also add new labels to indicate the area of issue:

Area|Possible Name 1|Possible Name 2|Possible Name 3|Includes
-|-|-|-|-
Nerdamer the library|(unlabeled)|library|nerdamer-library|nerdamer.core, unit tests, source code, etc.
Nerdamer the website|nerdamer.com|website|nerdamer-website|Documentation, demo, Getting Started etc.
Nerdamer the repository|meta|repo|nerdamer-repo|Wiki, Issues, pull requests, etc.

Maybe split them more into docs, demo, meta, source-code, unit-tests, etc. what do you think?

**Comment by jiggzson** [comment]
>Please change all the above Noes to Yeses.

Ok. I'll try to to get to this as soon as possible. 

>Maybe split them more into docs, demo, meta, source-code, unit-tests, etc. what do you think?

Sounds good. The current labels that Github provides don't cover many scenarios. I'm not sure you can create new ones. I know that Gitlab allows you to. It would be nice if we could do this. Maybe just a table then but then that requires maintenance.

**Comment by Happypig375** [comment]
Yes, you can create new labels. In the Labels tab under Issues, there is a green New Label button at the right.

**Comment by Happypig375** [comment]
Ummm... Still considered?

**Comment by jiggzson** [comment]
Documenting operators moved to this issue

**Comment by Happypig375** [comment]
Umm... I don't see the relation between this issue (organising other issues on GitHub) and #184 (Documenting operators).
